src/helix.idea.vim

```diff
@@ -0,0 +1,198 @@
+set wrapscan                      " searches wrap around the end of the file
+
+set ideajoin
+set idearefactormode=keep
+
+" Plugins
+set commentary
+set highlightedyank
+set argtextobj
+set matchit
+set multiple-cursors
+
+nnoremap , <Esc><Esc>
+xnoremap , <Esc><Esc>
+vnoremap , <Esc><Esc>
+
+xnoremap <Esc>o o
+nnoremap o o
+
+nnoremap d vd
+xnoremap d dv
+
+" Movement
+unmap <Left>
+unmap <Right>
+unmap <Down>
+unmap <Up>
+
+noremap t vt
+noremap T vT
+noremap f vf
+noremap F vF
+
+" Select/extend
+nnoremap e ve
+nnoremap E vE
+nnoremap w vw
+nnoremap W vW
+nnoremap b vb
+nnoremap B vB
+
+noremap X 0V
+xnoremap X <nop>
+
+" Match mode
+noremap ma) <Esc>va(
+noremap ma} <Esc>va{
+noremap ma> <Esc>va<
+noremap ma[ <Esc>va[
+
+noremap mi) <Esc>vi)
+noremap mi{ <Esc>vi}
+noremap mi< <Esc>vi>
+noremap mi[ <Esc>vi]
+
+" Goto mode
+nmap gr <Action>(FindUsages)
+nmap gi <Action>(GotoImplementation)
+nmap gd <Action>(GotoDeclaration)
+nmap gT <Action>(GotoTest)
+nmap gp <Action>(PreviousTab)
+nmap gy <Action>(GotoTypeDeclaration)
+
+" Requires plugin: KJump
+nmap gw <Action>(KJumpAction.Word0)
+xmap gw <Action>(KJumpAction.Word0)
+
+" Requires plugin: String manipulation
+xmap & <Action>(StringManipulation.AlignSelections)
+
+" Selection manipulation
+nmap <A-s> <Action>(EditorAddCaretPerSelectedLine)
+vmap <A-s> <Action>(EditorAddCaretPerSelectedLine)
+nmap <A-o> <Action>(EditorSelectWord)
+xmap <A-o> <Action>(EditorSelectWord)
+nmap <A-up> <Action>(EditorSelectWord)
+xmap <A-up> <Action>(EditorSelectWord)
+nmap <A-i> <Action>(EditorUnSelectWord)
+xmap <A-i> <Action>(EditorUnSelectWord)
+nmap <A-down> <Action>(EditorUnSelectWord)
+xmap <A-down> <Action>(EditorUnSelectWord)
+nmap C <Action>(EditorCloneCaretBelow)
+xmap C <Action>(EditorCloneCaretAbove)
+
+xnoremap ~ Ugv
+nnoremap ~ vU
+xnoremap ` ugv
+nnoremap ` vu
+
+nnoremap * v"ay:<C-u>let @/='<C-r>a'<CR>
+xnoremap * "ay:<C-u>let @/='<C-r>a'<CR>
+
+" View mode
+nmap zc <Action>(EditorScrollToCenter)
+nmap zj <Action>(EditorScrollDownAndMove)
+nmap zk <Action>(EditorScrollUpAndMove)
+
+" Unimpaired
+nmap ]p <Action>(EditorForwardParagraph)
+xmap ]p <Action>(EditorForwardParagraphWithSelection)
+nmap [p <Action>(EditorBackwardParagraph)
+xmap [p <Action>(EditorBackwardParagraphWithSelection)
+
+vmap ]f <Action>(MethodDown)
+nmap ]f <Action>(MethodDown)
+xmap [f <Action>(MethodUp)
+nmap [f <Action>(MethodUp)
+xmap ]d <Action>(GotoNextError)
+nmap ]d <Action>(GotoNextError)
+xmap [d <Action>(GotoPreviousError)
+nmap [d <Action>(GotoPreviousError)
+nmap ]g <Action>(VcsShowNextChangeMarker)
+nmap [g <Action>(VcsShowPrevChangeMarker)
+
+" Space mode
+nmap <Space>f <Action>(GotoFile)
+xmap <Space>f <Action>(GotoFile)
+nmap <Space>F <Action>(FindInPath)
+xmap <Space>F <Action>(FindInPath)
+
+nmap <Space>b <Action>(RecentFiles)
+xmap <Space>b <Action>(RecentFiles)
+nmap <Space>g <Action>(RecentChangedFiles)
+xmap <Space>g <Action>(RecentChangedFiles)
+
+nmap <Space>G <Action>(Debug)
```

Comment:
Maybe it's a good idea to gather all non-standard-Helix bindings in one spot, and comment it as such, so that people can easily see what's different from regular Helix

Reply:
I agree, I tried to make this as 1:1 to Helix defaults. I've got a separate config that overrides many mappings defined here. Debug seems to be default nowadays though:

![image](https://github.com/user-attachments/assets/55005235-94c6-45eb-b0a4-7ff9142829ee)

Reply:
Ah, my version doesn't have this one yet :)